### ISSUE TITLE
[now dev] @now/python builder exited with SIGKILL

### ISSUE THREAD

**Issue by lqmanh** [vercel/vercel#3416]
Hi,

I tried to run `now dev` on my Digital Ocean VPS and this error happened with Python lambdas. I don't know exactly why.

- OS Fedora 30
- `now` v16.6.3
- Debug log:
```
> [debug] [2019-12-12T11:38:03.970Z] Error: Builder exited with SIGKILL before sending build result
    at ChildProcess.onExit (/usr/lib/node_modules/now/dist/index.js:2:3462594)
    at ChildProcess.emit (events.js:203:15)
    at ChildProcess.EventEmitter.emit (domain.js:448:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

However, it doesn't happen with my Manjaro installation.

Do you know what may be the cause and how to fix it? Thank you!

**Comment by styfle** [comment]
Hi  @lqmanh 

Can you share the output when you run `now --debug -b NOW_BUILDER_DEBUG=1`?

**Comment by lqmanh** [comment]
Dear @styfle,
When I tried to deploy to Now, I encountered this error with my Python lambdas:
```
Error: The lambda function size (65.71mb) exceeds the maximum size limit (50mb). Learn more: https://zeit.co/docs/v2/deployments/concepts/lambdas/#maximum-bundle-size
```
As I found, there is currently no way to increase this `maxLambdaSize` beyond 50mb. That's why I try to run `now dev` on a remote server as a workaround.

**Comment by styfle** [comment]
Indeed, that is a different problem. We have made some changes to the platform so that error message is confusing, sorry about that.

We used to set the `maxLambdaSize` to 10 MB and allow configuring up to 50 MB.

Since you are already beyond 50 MB, it seems like you will need to remove dependencies.

If your project contains many static files (images, videos, etc) then this issue is probably related to #2830.

Can you share your source code?

**Comment by lqmanh** [comment]
> Since you are already beyond 50 MB, it seems like you will need to remove dependencies.

That's sad. I only need to use some basic dependencies like Sanic, Pillow and Numpy.

> Can you share your source code?

Sure. Here it is: [steganopy](https://github.com/lqmanh/steganopy). The Nuxt app can build successfully but the others always fail.

**Comment by styfle** [comment]
I was able to deploy your repository without errors here: https://steganopy.testing-internal.now.sh/api

When I visit the deployment inspector, I can click the Functions tab and see the size of each function.

None of your functions have exceeded the 50 MB limit.

![image](https://user-images.githubusercontent.com/229881/70736415-4427d500-1cde-11ea-9a06-1a7e761156fc.png)

**Comment by lqmanh** [comment]
That's because none of Python's dependencies was installed. `now-cli` only read them from `Pipfile.lock`, not `Pipfile` (I shouldn't have added it to .gitignore).

**Comment by styfle** [comment]
Hi @lqmanh We fixed a few issues in #3418 and #3712 

Can you trying deploying again to confirm this fixes your issue?

**Comment by lqmanh** [comment]
@styfle,
It works like a charm.
Thank you very much!
